### PR TITLE
feat: player movement in all cardinal directions

### PR DESCRIPTION
**Pull request by MrTanoshii**
#14

### CODE REVIEW

File: player.py [hunk 1]
```diff
@@ -119,3 +120,35 @@ def follow_mouse(self, mouse_x: float, mouse_y: float):
         else:
             new_angle = new_angle - SPRITE_PLAYER_INIT_ANGLE
         self.angle = new_angle
+
+    def update(self, move_dir):
```
Comment (mohith01) [inline player.py:129]: Zero divison breaking the game and probably limit the rotation @MrTanoshii

**Review by mohith01 — CHANGES_REQUESTED** [review]
Please check the changes

File: constants.py [hunk 2]
```diff
@@ -30,3 +30,15 @@ class level1:
 PLAYER_GUN_SHOOT_SPEED = 0.1
 PLAYER_GUN_RELOAD_TIME = 3
 PLAYER_GUN_DAMAGE = 1
```
Comment (BriscoRP) [inline constants.py:40]: row looks good!

**Review by BriscoRP — COMMENTED** [review]
all looked good.

File: player.py (hunk 1 above)
Comment (MrTanoshii) [reply player.py:129]: Fixed in 5cc198f71dd6bd96e94dab8f23e72ba5fd40089a

**Pull request closed without merging**